src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -110,6 +110,23 @@ def __publish_new_services(self, active_services: List):
         # save the cached services in the resolver.
         TopServiceResolver.store_cached_services(list(self.__services_cache.values()))
 
+    def __get_events_history(self):
```

Comment:
I would wrap this function in a try catch so if there is any unexpected errors here we will atleast run the sink